### PR TITLE
mm-CreateSBIPage

### PR DESCRIPTION
**Pull request by maya-maye**
In this PR, I added Search By Instructor Page, story, and test. Because the search by instructor form is not up yet, the page only has its title right now. 

Storybook entry: https://ucsb-cs156-s23.github.io/proj-courses-s23-7pm-2/prs/12/storybook/?path=/docs/pages-searchbyinstructor-searchbyinstructorpage--default

### CODE REVIEW

File: frontend/src/App.js
Comment (JNewman-cell) [inline frontend/src/App.js:1]: change professor to instructor like Justin's PR

File: frontend/src/main/pages/SearchByInstructor/SearchByInstructorPage.js
Comment (JNewman-cell) [inline frontend/src/main/pages/SearchByInstructor/SearchByInstructorPage.js:1]: Needs to set up functions to fetch courses by instructor using the backend function in main

**Review by JNewman-cell — CHANGES_REQUESTED** [review]

File: frontend/src/tests/pages/SearchByInstructor/SearchByInstructorPage.test.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+//       <QueryClientProvider client={queryClient}>
+//         <MemoryRouter>
+//           <SectionSearchesIndexPage />
+//         </MemoryRouter>
+//       </QueryClientProvider>
+//     );
+
+//     const selectQuarter = screen.getByLabelText("Quarter");
+//     userEvent.selectOptions(selectQuarter, "20222");
+//     const selectSubject = screen.getByLabelText("Subject Area");
```
Comment (pconrad) [inline frontend/src/tests/pages/SearchByInstructor/SearchByInstructorPage.test.js:64]: Commented out tests shouldn't go into a PR intended for merging into main.

Could you remove this test?

**Review by pconrad — CHANGES_REQUESTED** [review]
Please remove the commented out test.

**Review by JNewman-cell — APPROVED** [review]

**Comment by Arik** [comment]
https://github.com/ucsb-cs156-s23/proj-courses-s23-7pm-2/pull/34 further along

**Pull request closed without merging**